src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.data.tag;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.UniqueTagList.DuplicateTagException;
+
+public class Tagging {
+   private int status;
+   private Person person;
+   private UniqueTagList alltags;
```

Comment:
based on the diagram given, this should not be here. `Tagging` is just the association between a `Person` and a `Tag`